lib/forklift/box_distributor.rb

```diff
@@ -112,7 +112,8 @@ def configure_sshfs(config, box)
       config.vm.synced_folder box['sshfs']['host_path'],
                               box['sshfs']['guest_path'],
                               :type => :sshfs,
-                              :reverse => box['sshfs']['reverse'] || false
+                              :reverse => box['sshfs']['reverse'] || false,
+                              :sshfs_opts_append => box['sshfs']['sshfs_opts_append'] || ''
```

Comment:
Could we all this options instead? It'd match how I did the nfs synced folder linux__nfs_options above.

Reply:
you'd prefer it just be "options" and not "sshfs_opts_append"? i'm alright w/ that, give me the thumbs up and i'll edit PR to add to readme too

Reply:
Yea I think so, what do you think? It doesn't matter much to me but it'd make it consistent with the NFS synced folder we have which takes an 'options' parameter.